finch/processes/utils.py

```diff
@@ -507,15 +509,49 @@ def make_tasmin_tasmax_pairs(
         )
 
 
+def fix_broken_time_index(ds: xr.Dataset):
```

Comment:
What is this doing?  Does an .nc file have a 1850-1-1 datestamp?  I might have missed this as in most of my scripts I select time>=1950  before doing anything

Reply:
Should we just fix this on disk? ... i.e. rewrite that file with a corrected timestamp instead of having a 'correction' function in finch?

Reply:
Just read the description of the PR re: the problematic file ... sorry

Reply:
I agree, it should probably be fixed on disk...

``` python
>>> import xarray as xr
>>> ds = xr.open_dataset('https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/birdhouse/pcic/BCCAQv2/tasmax_day_BCCAQv2+ANUSPLIN300_CESM1-CAM5_historical+rcp85_r1i1p1_19500101-21001231.nc')
>>> ds.time[31688:31693].values
array([cftime.DatetimeNoLeap(2036-10-26 12:00:00), 
       cftime.DatetimeNoLeap(2036-10-27 12:00:00),
       cftime.DatetimeNoLeap(1850-01-01 00:00:00),
       cftime.DatetimeNoLeap(2036-10-29 12:00:00),
       cftime.DatetimeNoLeap(2036-10-30 12:00:00)], dtype=object)
>>> ds.time[31690].values 
array(cftime.DatetimeNoLeap(1850-01-01 00:00:00), dtype=object)
```

Here I'm trying to fix it on the fly, with as less overhead as possible.

Reply:
I guess leave it as is for now... we can fix the file when we have time then remove or not this subfunction as desired